fluid_onnx/ops.py

```diff
@@ -224,8 +252,63 @@ def lppool_op():
     pass
 
 
-def matmul_op(inputs, attrs, outputs):
-    return make_node('MatMul', inputs=inputs, outputs=outputs)
+def mul_op(operator, scope):
```

Comment:
As we can see, sometimes we can't implement the operator in Fluid by only operator in ONNX. So it would be better to make the function name (`mul_op` here) consistent with the op's name in Fluid ('mul').